portfolio/src/main/webapp/comments.html

```diff
@@ -13,7 +13,7 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body onload="commentsPageOnLoad()">
+  <body onload="setupCommentsPage()">
```

Comment:
Nit: How about "loadCommentsPage"? 
I say this because you're actually filling up the content of the page, and not just setting up (with content loading elsewhere).

Reply:
Sounds good